### PR TITLE
Fix 482

### PR DESCRIPTION
**Pull request by vinipsmaker**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/crust/484)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @inetic

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/util.rs [hunk 1]
```diff
@@ -133,6 +133,36 @@ pub fn is_local(ip_addr: &IpAddr, interfaces: &[Interface]) -> bool {
...
 
+fn ip(addr: &net::SocketAddr) -> IpAddr {
+    match *addr {
+        net::SocketAddr::V4(ref addr) => IpAddr::V4(*addr.ip()),
+        net::SocketAddr::V6(ref addr) => IpAddr::V6(*addr.ip()),
+    }
+}
+
+/// If the endpoint IP address is unspecified return a copy of the endpoint with the IP address
+/// set to the loopback address. Otherwise return a copy of the endpoint.
```
Comment (dirvine) [inline src/util.rs]: This should be config test AFAIK as it's only used for the test function. All in all I feel it needs removed though.

Comment (vinipsmaker) [inline src/util.rs]: https://github.com/maidsafe/crust/pull/484/files#diff-f7206c90a05dcea1bf538304ccae0949R109 is production code, not test code.

**Pull request merged**